src/style.rs

```diff
@@ -238,7 +238,7 @@ define_styles!(
     cursor nocb: Option<CursorStyle> = None,
     color nocb: Option<Color> = None,
     background nocb: Option<Color> = None,
-    font_size nocb: Option<f32> = None,
+    font_size nocb: f32 = 14.0,
```

Comment:
With ```Option<f32>```, I could set a font size at the parent, and the children will have the same font size. With this, the default style will override the parent's font size.

Reply:
Oof, I see. I will change it back then.